atproto-openapi-types/main.ts

```diff
@@ -101,15 +101,15 @@ const api: OpenAPIV3_1.Document = {
   },
   servers: [
     {
-      url: "https://public.api.bsky.app/xrpc/",
+      url: "https://public.api.bsky.app/xrpc",
       description: "Bluesky AppView (Public, No Auth)",
     },
     {
-      url: "https://pds.example.org/xrpc/",
+      url: "https://pds.example.org/xrpc",
       description: "Example atproto PDS (Authenticated)",
     },
     {
-      url: "https://bsky.network/xrpc/",
+      url: "https://bsky.network/xrpc",
```

Comment:
This change fixes a display bug where now that all of the paths have the `/` at the beginning, the oai explorer lib was showing the full URL with double `//` in the `MethodEndpoint` component

<img width="548" alt="image" src="https://github.com/bluesky-social/bsky-docs/assets/9257001/47cb3cd6-ab47-4b25-a1d1-6b7c8749c5e5">